plugins/woocommerce-admin/client/shipping/experimental-shipping-recommendations.tsx

```diff
@@ -49,6 +49,13 @@ const ShippingRecommendations: React.FC = () => {
 		};
 	} );
 
+	if (
+		activePlugins.includes( 'woocommerce-shipping' ) ||
+		activePlugins.includes( 'woocommerce-tax' )
+	) {
+		return <ShippingTour showShippingRecommendationsStep={ false } />;
```

Comment:
This prevents this banner from being displayed on the WC > Settings > Shipping page if WC Shipping or WC Tax is active:

<img width="722" alt="Screenshot 2024-06-06 at 13 31 02" src="https://github.com/woocommerce/woocommerce/assets/1759681/177f8ff6-7957-4368-a608-195f0d9718de">

The banner mentions "WooCommerce Shipping" but what it installs is the WCS&T extension.